packages/nimble-components/src/banner/template.ts

```diff
@@ -35,13 +35,13 @@ export const template = html<Banner>`
     >
         <div class="icon">
             ${when(x => x.severity === BannerSeverity.error, html`
-                <${iconExclamationMarkTag}></${iconExclamationMarkTag}>
+                <${iconExclamationMarkTag} role="img" aria-label="Error"></${iconExclamationMarkTag}>
```

Comment:
I think we should offer apps a way of localizing these labels by setting them via the locale provider instead of hard-coded strings.

Reply:
Good point. Added.